### PR TITLE
Add back Shared/persistent runner scripts

### PR DESCRIPTION
**Pull request by yuennancy**
These scripts are used to install and maintain persistent self hosted runners such as the A100 GCP VMs and the runners for Pixel phones.

The scripts were on the lab-runner branch, but were deleted in a branch cleanup effort.

### CODE REVIEW

File: build_tools/github_actions/runner/config/runner.env [hunk 1]
```diff
@@ -1,3 +1,4 @@
 LANG=C.UTF-8
+RUNNER_GROUP=presubmit
```
Comment (ScottTodd) [inline build_tools/github_actions/runner/config/runner.env:2]: Should this be hardcoded to presubmit? Is this also used for postsubmit runners?

**Review by ScottTodd — APPROVED** [review]
Did you want to merge these changes to `main`, or keep them on `shared/persistent-runner`?

File: build_tools/github_actions/runner/config/runner.env (hunk 1 above)
Comment (yuennancy) [reply build_tools/github_actions/runner/config/runner.env:2]: Yes, when this was used in the postsubmit runner, it was modified during the install.  I saw some notes in the install doc (internally) that instruct to change the configuration to match the runner type.

I think these scripts were written for presubmit runners but they adapted to for postsubmit.  But the scripts didn't get updated and they probably should be.

**Comment by yuennancy** [comment]
> Did you want to merge these changes to `main`, or keep them on `shared/persistent-runner`?

I'm not sure the changes to the existing files will work with the existing scripts that are being used by the ephemeral runners.  They're small changes but I'd need another look at the ephemeral runner scripts.  I don't want to break the ephemeral runners, they're the bulk of the runners being used.  For now, would prefer to keep them on the branch.

**Review by jpienaar — APPROVED** [review]

**Comment by jpienaar** [comment]
Anything else needed here?

**Pull request closed without merging**